net.sourceforge.qtpfsgui.LuminanceHDR.yaml

```diff
@@ -0,0 +1,110 @@
+app-id: net.sourceforge.qtpfsgui.LuminanceHDR
+runtime: org.kde.Platform
+runtime-version: "5.11"
+sdk: org.kde.Sdk
+command: luminance-hdr
+rename-icon: luminance-hdr
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=home
+  - --share=network
+  - --device=dri
+modules:
```

Comment:
I suspect everything could use some cleanups of development headers etc.

Reply:
@bilelmoussaoui regarding this, is `flatpak-builder run...` sufficient to look what's inside the built container? Or there is better option for this?

Reply:
Yes, or you can inspect `builddir/files` directory.

Reply:
What @barthalion suggested is perfect for this. A huge + if you can split the cleanup per module and put the common ones on the top cleanup, it's not a requirement, just a suggestion :)